common/src/main/kotlin/entity/DiscordMessage.kt

```diff
@@ -264,7 +264,7 @@ data class DiscordMentionedChannel(
         val name: String,
 )
 
-enum class MessageFlag(val code: Long) {
+enum class MessageFlag(val code: Int) {
```

Comment:
Why the change? Why longs in the first place?

Reply:
This was due to a typo in  https://github.com/discord/discord-api-docs/pull/2615
which had Empheral flag set as (1 << 64) but I was actually (1 << 6). (The post has been edited since we pointed this out)
The commit was reverted later on in the branch back to int

Reply:
This sounds like something that might grow in the future, like the permission flags